### DISCUSSION TITLE
le build type

### DISCUSSION THREAD

**Discussion by MangoIV** [flora-pm/flora-server#512] [Feature Requests]
it would be nice to have the cabal build type displayed :)

**Comment by tchoutri** [comment]
Where do you reckon we could put that information? 

![image](https://github.com/flora-pm/flora-server/assets/15848443/1ecb1803-d344-4a44-8b9d-2b7afe55e7b9)

![Screenshot 2024-02-25 at 15-42-34 @haskell_text](https://github.com/flora-pm/flora-server/assets/15848443/8bc16393-fa53-48d6-ae2a-3a345d07d540)

I think I have a preference for the first one, using a badge.

**Reply by MangoIV** [reply]
Yeah use a badge but also make it red vs green 😌

**Reply by MangoIV** [reply]
Where red means custom

**Reply by tchoutri** [reply]
hmm not sure about the colour-coding. Red is used for deprecation in the rest of the package page. At the very least, orange for "warning" can be used. I don't also want to use space for Simple build type because it's the default for many things, and it's not very remarkable.

**Reply by MangoIV** [reply]
Thank you!
